packages/core/core/src/PackagerRunner.js

```diff
@@ -398,9 +398,14 @@ export default class PackagerRunner {
     let {name, resolveFrom, plugin} = packager;
     let measurement;
     try {
-      measurement = tracer.createMeasurement(name, 'packaging', bundle.name, {
-        type: bundle.type,
-      });
+      measurement =
+        tracer.enabled &&
```

Comment:
I know moving this check to the consumer avoids the function call at all, but is the only reason to change the `create` signature from individual params to an object? I'm still not sure what the benefit is apart from the naming?

Reply:
It's the naming and flexibility mostly, yeah